### PR TITLE
Improve DATA.md

### PR DESCRIPTION
**Pull request by outdooracorn**
* Make it clearer in the ASCII diagram what actions are specific to the old data and what are specific to the new data
* Add relevant hyperlinks
* Fix typo

### CODE REVIEW

File: docs/data/DATA.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Data
 
-Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster. See DATA-NEW.md
+Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster, see [DATA-NEW.md](DATA-NEW.md).
 
-Old data can also be gathered from the old wikidata-analysis project, See DATA-OLD.md
+Old data can also be gathered from the old [wikidata-analytics]([url](https://github.com/vrandezo/wikidata-analytics)) project, see [DATA-OLD.md](DATA-OLD.md).
```
Comment (outdooracorn) [inline docs/data/DATA.md:5]: Is this the correct project?

Comment (addshore) [reply docs/data/DATA.md:5]: I think so

Comment (addshore) [reply docs/data/DATA.md:5]: Wait, no, not in the cpontext of this sentence.
This github repo is the very very very old origins of this whole code and idea.
wikidata-analytics is the wmflabs project / tool
I dont appear to be able to find a link to it currently.

Comment (outdooracorn) [reply docs/data/DATA.md]: Okay, I've removed that link then

File: docs/data/DATA.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # Data
 
-Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster. See DATA-NEW.md
+Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster, see [DATA-NEW.md](DATA-NEW.md).
 
-Old data can also be gathered from the old wikidata-analysis project, See DATA-OLD.md
+Old data can also be gathered from the old wikidata-analysis project, see [DATA-OLD.md](DATA-OLD.md).
```
Comment (addshore) [inline docs/data/DATA.md:5]: @outdooracorn The tool is actually linked to from the linked to documentation https://github.com/wmde/wikidata-map/blob/master/docs/data/DATA-OLD.md

**Pull request merged**